.travis.yml

```diff
@@ -34,10 +34,10 @@ script:
   - cmake -DDEFINE_DEBUG=ON -DLINUX_BUILD=ON -DTRAVIS_BUILD=ON -DCMAKE_CXX_FLAGS=" -fsanitize=leak -pedantic -Wall -Wextra -Wnon-virtual-dtor -Werror -Wold-style-cast -Wcast-align -Wunused -Woverloaded-virtual -Wpedantic -Wconversion -Wnull-dereference -Wdouble-promotion -Wduplicated-cond -Wduplicated-branches -Wlogical-op -g" -DCMAKE_C_FLAGS=" -fsanitize=leak"  -DCMAKE_EXE_LINKER_FLAGS=" -fsanitize=leak -pedantic -Wall -Wextra -Wnon-virtual-dtor -Werror -Wold-style-cast -Wcast-align -Wunused -Woverloaded-virtual -Wpedantic -Wconversion -Wnull-dereference -Wdouble-promotion -Wduplicated-cond -Wduplicated-branches -Wlogical-op -g" -DCMAKE_MODULE_LINKER_FLAGS="  -fsanitize=leak  -pedantic -Wall -Wextra -Wnon-virtual-dtor -Werror -Wold-style-cast -Wcast-align -Wunused -Woverloaded-virtual -Wpedantic -Wconversion -Wnull-dereference -Wdouble-promotion -Wduplicated-cond -Wduplicated-branches -Wlogical-op -g" ../
   - make -j 12
   - cd bin
-  - ./runUnitTests
+  - for i in `ls`; do echo "Running -> " $i; ./$i; echo $i "ended."; done
```

Comment:
There is a weird \n after event_time_test. Please check it and fix it. Also, let's stick to one formatting i.e.
Running -> elf
Ending -> elf
Other stuff looks fine :)

Reply:
:)